include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -165,7 +198,9 @@ class CommonOperatorLoader : public ProtobufLoader {
       node = G_.createSub(opName, in0, finalIn1);
     } else if (typeName == "Div") {
       node = G_.createDiv(opName, in0, finalIn1);
-    } else {
+    }
```

Comment:
Please undo this spacing change -- clang-format should fix it.

Reply:
Done